### ISSUE TITLE
User Warning

### ISSUE THREAD

**Issue by Carinazhao22** [DanJun6737/TransFace#6]
Hi,

When I tried to train the model with customized datasets, it shows this user warning. 
' UserWarning: None of the inputs have requires_grad=True. Gradients will be None   warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")' How can I fix this warning? Will it affect the model performance? Thanks!

**Comment by DanJun6737** [comment]
Hi @Carinazhao22 

I did not encounter this problem when training the model on the default dataset （e.g., MS1MV2 and Glint360K).  I suggest you check if your data has been successfully processed through the data preprocessing program, especially the part where image data is converted into tensors. 

You can train for a few epochs and then check the model's accuracy on the validation sets (e.g., LFW, CFP-FP and AgeDB-30) to determine whether this warning would affect the model's performance. 

If you need further assistance, it would require you to provide specific error details.

**Comment by DanJun6737** [comment]
@Carinazhao22 
Are you using a checkpoint? I googled and found that this warning could potentially be caused by incorrect usage of a checkpoint.
